### ISSUE TITLE
In line  89 'loss_and_miner_utils.py', is 'torch.where(triplets)' right?  torch.where() needs three parameters

### ISSUE THREAD

**Issue by long-username** [KevinMusgrave/pytorch-metric-learning#214]
Firstly, thank you for your code, it helps me a lot. 

in `loss_and_miner_utils.py` there is a function `get_all_triplets_indices(labels, ref_labels=None)` whose last statement ,`torch.where(triplets)` needs three parameters, but only one here. I think it should be `torch.nonzero(triplets).transpose(-1,0)`, is it right.

**Comment by KevinMusgrave** [comment]
Check out the documentation: https://pytorch.org/docs/stable/generated/torch.where.html

If you scroll down, you'll see there's a second version of ```torch.where``` which takes in a single argument.

**Comment by long-username** [comment]
Thank you very much. 
My pytorch needs to upgrade 👍 :)
